README_LINUX.md

```diff
@@ -31,7 +31,7 @@ Recommended packages
 
 For sclang and scide:
 
-- [Qt][Qt] >= 5.7 with QtWebEngine and QtWebSockets: Cross-platform GUI library, required for the IDE and for sclang's Qt GUI kit. It's best to get the latest Qt 5.x version.
+- [Qt][Qt] >= 6.2 with QtWebEngine and QtWebSockets: Cross-platform GUI library, required for the IDE and for sclang's Qt GUI kit. It's best to get the latest Qt 5.x version.
```

Comment:
> It's best to get the latest Qt 5.x version.

Is this still the case?

Reply:
Good catch! This PR is still very much WIP.